packages/types/src/schemas.ts

```diff
@@ -0,0 +1,3280 @@
+export const PeriodTypeSchema = {
```

Comment:
As long as EntityType is imported from '../../models' in types, all types in models will be converted too. Can't find a solution to avoid that.